neofetch

```diff
@@ -788,7 +788,7 @@ image_source="auto"
 #       Chapeau, Chrom*, Cleanjaro, ClearOS, Clear_Linux, Clover, Condres,
 #       Container_Linux, Crystal Linux, CRUX, Cucumber, dahlia, Debian, Deepin,
 #       DesaOS, Devuan, DracOS, DarkOs, Itc, DragonFly, Drauger, Elementary,
-#       EndeavourOS, Endless, EuroLinux, Exherbo, Fedora, Feren, FreeBSD,
+#       EndeavourOS, EncryptOS Endless, EuroLinux, Exherbo, Fedora, Feren, FreeBSD,
```

Comment:
I think there should be a comma here 🔍

Reply:
Also, is EncryptOS an actual distro? I can't find anything online...

Reply:
> Also, is EncryptOS an actual distro? I can't find anything online...

Yeah, it is an unreleased distro at the current time.